mitodl/ocw-studio
# Remove the internal external radio buttons

### What are the relevant tickets?
Closes https://github.com/mitodl/hq/issues/4101

### Description (What does it do?)
It removes the Item type radio button in `Menu Item Form`.
<img width="1680" alt="Screenshot 2024-05-27 at 5 52 27 AM" src="https://github.com/mitodl/ocw-studio/assets/71461724/dc9251bd-14fc-4a5a-a353-8ed41625ef52">


### How can this be tested?
1. Login to `cow-studio`
2. Add a new site course or use and existing one
3. In the course setting, visit page for `Menu`
<img width="301" alt="Screenshot 2024-05-27 at 5 55 10 AM" src="https://github.com/mitodl/ocw-studio/assets/71461724/7ee9d9e2-5486-4646-87b3-b9c84fce8dae">

4. Add a new menu item or open an existing one
5. The item form should not have radio button as shown in the above picture in description
